<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>793</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>483</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Contact Manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/icon.jpeg</normaloff>../img/icon.jpeg</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QScrollArea" name="contacts_scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="contacts_scrollAreaW">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>463</width>
         <height>597</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="contacts_layout">
        <item>
         <widget class="QLabel" name="contact0_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>PushButton</string>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="7" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item row="0" column="0" alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="newContact_pb">
        <property name="text">
         <string>New contact</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../img/addContact.png</normaloff>../img/addContact.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Sort by:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" alignment="Qt::AlignLeft">
       <widget class="QComboBox" name="tags_combo_box">
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="search_line_edit">
        <property name="placeholderText">
         <string>Search...</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="sort_combobox">
        <property name="currentText">
         <string>Name</string>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAtCurrent</enum>
        </property>
        <item>
         <property name="text">
          <string>Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Surname</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="tags_filter_label">
        <property name="text">
         <string>Filter  tag:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QToolButton" name="search_pb">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset theme="search">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>483</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
